### DISCUSSION TITLE
Consent Form

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/methods#68] [Ideas]
Hello! Might we have a template for a user research participant consent form? Something that easily accepts a digital signature of some kind would be ideal. Like a PDF. I am working on one for my thesis and am happy to provide when its done if that's helpful. Or, if one objects to PDFs, maybe just some re-usable language and guidance on ways to get a digital signature. Just thinking that's probably valuable in pandemic-times. (Submitted by: katharine.carr@bixal.com)

**Comment by pglevy** [comment]
Thanks for the input, @katharine-carr! One of the issues we're tackling this cycle is inventorying what's already hiding deep within the Methods. There actually is a "[participant agreement](https://bixal.github.io/methods/participant-agreement/)," but it's only linked from the [Interview Checklist](https://bixal.github.io/methods/interview-checklist/), which is linked from the [Stakeholder and User Interviews](https://bixal.github.io/methods/awareness/stakeholder-and-user-interviews/) method. So we're working on surfacing and re-assessing those. 

You raise good points about the PDFs and digital signature use cases. What do you think of this participant agreement? Is it missing anything? Have something not relevant to us? Let's definitely capture any feedback or work from your thesis here so we can incorporate any changes into a future cycle.

**Comment by pglevy** [comment]
Good reference from Katharine: https://www.userfocus.co.uk/articles/anatomy-of-a-consent-form.html

**Comment by katharine-carr** [comment]
Coming back to this now that I've finished my consent forms and everything has been officially approved by the review board! For reference, I'm attaching the consent form I'll be using in this research study, the template from UBalt (which isn't that bad if you can see past the color choices), their guidance on required elements of informed consent, and a consent form template I pulled from usability.gov. 


[usability.gov_consent-form-adult.docx](https://github.com/Bixal/methods/files/5815457/usability.gov_consent-form-adult.docx)
[UBalt_Updated_Consent_Form.docx](https://github.com/Bixal/methods/files/5815458/UBalt_Updated_Consent_Form.docx)
[UBalt_Required_Elements_of_Informed_Consent.docx](https://github.com/Bixal/methods/files/5815459/UBalt_Required_Elements_of_Informed_Consent.docx)
[KCarr_IDIAThesis_REVISED Research Study Consent Form_Interviews.pdf](https://github.com/Bixal/methods/files/5815460/KCarr_IDIAThesis_REVISED.Research.Study.Consent.Form_Interviews.pdf)

**Reply by pglevy** [reply]
Cool, thanks. I'll take a look at these and think about whether we want to try to address something with this next cycle.
